webapp/components/interop.html

```diff
@@ -235,12 +251,16 @@
         },
         searchResults: Array,
         onSearchCommit: Function,
+        interopLoadFailed: Boolean,
       };
     }
 
     constructor() {
       super();
       this.onSearchCommit = this.handleSearchCommit.bind(this);
+      this.onLoadingComplete = () => {
+        this.interopLoadFailed = !(this.passRateMetadata && this.allMetrics);
```

Comment:
It's not immediately obvious why `&& this.allMetrics` is included. Maybe add a comment about this predicate?

Reply:
The metadata is the runs and where to find the interop JSON blob, the allMetrics is the fetched JSON blob. If either fails, then the view is borked.